nvdaHelper/vbufBackends/mshtml/mshtml.cpp

```diff
@@ -1140,7 +1163,33 @@ if(!(formatState&FORMATSTATE_INSERTED)&&nodeName.compare(L"INS")==0) {
 		attribsMap.find(L"HTMLAttrib::aria-label") != attribsMap.end() || attribsMap.find(L"HTMLAttrib::aria-labelledby") != attribsMap.end()
 		|| attribsMap.find(L"HTMLAttrib::title") != attribsMap.end() || attribsMap.find(L"HTMLAttrib::alt") != attribsMap.end()
 	)))
-		attribsMap[L"name"]=IAName;
+		{attribsMap[L"name"]=IAName;

```

Comment:
Place the left brace ({) on the line above to conform with  code formatting for the rest of the file.